packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -455,9 +456,25 @@ export interface IPerfSignalReport {
 	signalTimestamp: number;
 
 	/**
-	 * Expected Signal Sequence to be received.
+	 * Signal we will trace for roundtrip latency.
+	 */
+	roundTripSignalSequenceNumber: number | undefined;
+
+	/**
+	 * Last signal traced for roundtrip latency.
+	 */
+	previousRoundTripSignalSequenceNumber: number | undefined;
+
+	/**
+	 * Next expected signal sequence number to be received.
+	 * Inclusive upper bound of signal monitoring window.
 	 */
 	trackingSignalSequenceNumber: number | undefined;
```

Comment:
Is that true? That it is upper bound of the monitoring window? Aren't we really monitoring all numbers above the min?
(We aren't going to ignore things higher, right?)

Reply:
Yes, it's not really an upper-bound since we don't ignore higher seq numbers. Removed this line and I think "Next expected signal sequence number to be received." is sufficient.